src/index.js

```diff
@@ -154,6 +153,17 @@ class App extends React.Component {
          <button onClick={ () => this.handleChangeLoadMediaUrl() }>
           Load Media From Url
          </button>
+         <br/>
+         <label>open Transcript Json</label>
+         <SttTypeSelect
+           name={ 'sttType' }
+           value={ this.state.sttType }
+           handleChange={ this.handleSttTypeChange }
+         />
+         <input
+           type="file"
+           onChange={ e => this.handleChangeLoadTranscriptJson(e) }
```

Comment:
does this need the `e` passed in? Or does ` onChange={ this.handleChangeLoadTranscriptJson }` work?

Reply:
Tried removing the `e` and it crashes.